### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by brendan-gero-humanetix**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -73,5 +73,8 @@
     "src"
   ],
   "license": "MIT",
-  "sideEffects": false
+  "sideEffects": false,
+  "dependencies": {
```
Comment (tomsaunders) [inline package.json]: Was this just for your benefit to get things running or is it now required for everyone?

File: src/datetime.js [hunk 2]
```diff
@@ -160,15 +160,23 @@ function adjustTime(inst, dur) {
 
 // helper useful in turning the results of parsing into real dates
 // by handling the zone options
-function parseDataToDateTime(parsed, parsedZone, opts, format, text) {
+function parseDataToDateTime(parsed, parsedZone, parsedOffsetZone, opts, format, text) {
+  // console.log("parseDateToDateTime", parsed, parsedZone, opts, format, text);
```
Comment (tomsaunders) [inline src/datetime.js]: log

File: src/datetime.js [hunk 3]
```diff
@@ -160,15 +160,23 @@ function adjustTime(inst, dur) {
 
 // helper useful in turning the results of parsing into real dates
 // by handling the zone options
```
Comment (tomsaunders) [inline src/datetime.js]: adding a comment would probably be good, particularly to distinguish between `parsedZone` and `parsedOffsetZone`

File: test/datetime/tokenParse.test.js [hunk 4]
```diff
@@ -18,6 +18,124 @@ test("DateTime.fromFormat() parses basic times", () => {
...
+  expect(i.hour).toBe(9);
+  expect(i.minute).toBe(7);
+  expect(i.second).toBe(13);
+  expect(i.millisecond).toBe(0);
+  expect(i.offset).toBe(480); //+08:00
+  expect(i.zoneName).toBe("Australia/Perth");
+});
+
+test("DateTime.fromFormat() converts offset not belonging to time zone", () => {
+  // +11:00 is not a valid offset for the Australia/Perth time zone
```
Comment (tomsaunders) [inline test/datetime/tokenParse.test.js:40]: I would question whether anything incorrect is supposed to silently become correct. Is there a test for handling February 30 ?

**Review by tomsaunders — COMMENTED** [review]
submitted review

File: test/datetime/tokenParse.test.js (hunk 4 above)
Comment (brendan-gero-humanetix) [reply test/datetime/tokenParse.test.js:40]: Judging from this reply to my question, I think that's the route the maintainer wants to go from now on: https://github.com/moment/luxon/issues/1076#issuecomment-966015985

February 30th is an interesting case though.

File: package.json (hunk 1 above)
Comment (brendan-gero-humanetix) [reply package.json]: Reverted this. It seems to have occurred from following instructions to get tests working. Not sure why it is not committed in the first place.

File: src/datetime.js (hunk 2 above)
Comment (brendan-gero-humanetix) [reply src/datetime.js]: Removed

File: src/datetime.js (hunk 3 above)
Comment (brendan-gero-humanetix) [reply src/datetime.js]: Done

**Review by tomsaunders — APPROVED** [review]
I think you can submit this to the real repo

**Pull request closed without merging**